integreat_cms/cms/migrations/0103_userchat_language_userchat_region_and_more.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 4.2.13 on 2024-09-19 06:36
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+from ...cms.models.regions.region import Region
+
+class Migration(migrations.Migration):
+    """
+    Add relation between user chats, region and language
+    """
+    dependencies = [
+        ("cms", "0102_alter_contact_poi"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="userchat",
+            name="language",
+            field=models.ForeignKey(
+                default=3,
```

Comment:
All existing chat which were created during the first test phase, will be assigned the German language. There is no other good solution.